src/constants/text.js

```diff
@@ -11,6 +11,18 @@ const showEvents = (numberOfEvents: number) => {
   return `Show ${numberOfEvents} events`;
 };
 
+const showEventsLabel = (numberOfEvents: number) => {
```

Comment:
These labels are very similar to the normal ones. I wonder if we should speak to UX to get one unified text instead?

Reply:
The reader one are quite a lot longer to be very descriptive, since you lack the visual context, I think the visual ones are fine like this... but we can have a chat